shared/src/main/scala/amf/core/internal/parser/AMFCompiler.scala

```diff
@@ -193,8 +195,13 @@ class AMFCompiler(compilerContext: CompilerContext, val referenceKind: Reference
     Future.sequence(parsed).map(rs => root.copy(references = rs.flatten))
   }
 
-  private[amf] def fetchContent()(implicit executionContext: ExecutionContext): Future[Content] =
-    compilerContext.fetchContent()
+  private[amf] def fetchContent()(implicit executionContext: ExecutionContext): Future[Content] = {
+    compilerContext.fetchContent().map { c =>
+      if (isRoot && c.stream.currentChar() > 0x7F)
+        c.copy(stream = new CharSequenceStream(c.stream.toString.substring(1)))
```

Comment:
Thought: this could easily be implemented with a Resource Loader, why didn't we do it there or why didn't we tell RestSDK to do it there? (Without taking into account usability)